octarine/supported_ops.go

```diff
@@ -34,20 +33,22 @@ const (
 
 var supportedOps = map[string]supportedOperation{
 	installOctarineCommand: {
-		name: "Install the latest version of Octarine's data plane",
+		name: "Install latest version of Octarine's data plane",
```

Comment:
Can we please change this to just "Latest version of Octarine's data plane" ? That way it will be neutral in both install and remove cases.